care/facility/models/patient.py

```diff
@@ -432,6 +432,10 @@ class TestTypeEnum(enum.Enum):
         AbhaNumber, on_delete=models.SET_NULL, null=True, blank=True
     )
 
+    has_consents = models.BooleanField(
```

Comment:
consents are consultation specific 
wouldn't it be better to have this on consultation